numpy/lib/tests/test_stride_tricks.py

```diff
@@ -325,6 +326,60 @@ def test_as_strided():
     assert_equal(a.dtype, a_view.dtype)
     assert_array_equal([r] * 3, a_view)
 
+
+def test_sliding_window_view():
+    arr = np.array([None])
+    shape = [1]
+    arr_view = sliding_window_view(arr, shape)
+    expected = np.array([None])
+    assert_array_equal(arr_view, expected)
+
+    arr = np.arange(12).reshape(3,4)
```

Comment:
A more intuitive array for understanding the result is

```
i, j = np.ogrid[:3,:4]
arr = 10*i + j
```

Since then one digit is columns, and the other is rows

Reply:
fixed